server.go

```diff
@@ -180,12 +180,13 @@ func (s *Server) Routes() []Route {
 	return s.routes
 }
 
-func (s *Server) ConfigureTracing(endpoint string, serviceName string, serviceVersion string, insecure bool) {
```

Comment:
non-blocking, but it could be nice to make this accept a config struct at some point in the future.